src/components/scenes/SendScene.js

```diff
@@ -181,7 +172,9 @@ class SendComponent extends React.PureComponent<Props, State> {
     const { props } = this
```

Comment:
Change this to `const { route } = this.props`, and then do `const { allowedCurrencyCodes } = route.params` on the next line.